source/includes/fact-older-version-limitations.rst

```diff
@@ -0,0 +1,42 @@
+- Writes that produce DDL events cannot occur on the source cluster during the 
+  migration. The following events cannot occur: 
+  
+  - ``create``
+  - ``drop``
+  - ``dropDatabase``
+  - ``createIndexes``
+  - ``dropIndexes``
+  - ``collModrefineCollectionShardKey``
+  - ``rename``
+  - ``reshardCollection``
+  - ``shardCollection``
+
+  This includes operations that may create new collections such as 
+  :dbcommand:`mapReduce`, :pipeline:`$out`, and :pipeline:`$merge`. This also 
+  includes collections created implicitly from inserts. Only writes that produce 
+  CRUD events can occur during the migration.
+
+  .. note:: 
+   
+     Writes that produce DDL events on source collections outside of the 
+     :ref:`namespace filter <c2c-filtered-sync>` are allowed.
+
+- ``geoHaystack`` indexes are not supported.
+
+- :ref:`/reverse <c2c-api-reverse>` endpoints are not supported. You can't 
```

Comment:
Are you planning to update the reverse limitation on the reverse page in a separate PR?